unit1/lesson14.rst

```diff
@@ -0,0 +1,28 @@
+
+Unit 1 - Lesson 14 Learning Objectives
+======================================
+
+Goals
+-----
+
+- Understand the DOM is used to make dynamic web pages
+- Use DOM objects and methods to alter an HTML web page
+- Understand what events are and that JavaScript is an event driven language
+- Handle and trigger events in JavaScript
+
+Objectives
+----------
+
+- Know what the DOM acroynym stands for
+- Understand what a Document Object Model is
+- Know how the DOM relates to HTML, CSS, and the browser
+- Differentiate between dynamic and static web pages
+- Describe how the DOM is used to make web pages dynamic
+- Understand why the DOM is written in JavaScript and lives in the browser
+- Know that JavaScript is an event driven programing language
+- Explain what an event is
+- Trigger and handle events
```

Comment:
Clarifying suggestion: Break this into multiple objectives, and be more explicit.

- Write an event handler function
- Use arguments provided to event handlers to access information about the triggering event
- Explicitly trigger an event using code

Reply:
Will do.